ffc-helm-library/templates/_ingress.yaml

```diff
@@ -1,6 +1,6 @@
 {{- define "ffc-helm-library.ingress.tpl" -}}
 {{- $requiredMsg := include "ffc-helm-library.default-check-required-msg" . -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
AWS deploys will be using this ingress template so you need to check this `apiVersion` is installed on the AWS K8s cluster.

Reply:
I'll revert the change to be on the safe side.